src/lib/vars.ts

```diff
@@ -1,11 +1,16 @@
+import { getFigmaLinks } from './getFigmaLinks'
+
 export * from './cookieSettings'
 
 export const baseUrl = process.env.BASE_URL
   ? process.env.BASE_URL.replaceAll(/\/$/g, '')
   : 'https://design-system.service.mod.gov.uk'
 export const contactEmail = 'design-system@digital.mod.uk'
-export const figmaLink =
-  'https://www.figma.com/file/JrmVrSdKCHUZBQFEB3xsjJ/Design-Library?node-id=50%3A5303&t=xG6UMcIDeLIbH1kD-1'
+export const figmaLinks = getFigmaLinks({
```

Comment:
Maybe `figmaURLs` (rather than `figmaLinks`) would be a bit more specific about the values being URLs?